draft-fossati-tls-exported-attestation.md

```diff
@@ -72,38 +86,119 @@ There is a growing need to demonstrate to a remote party that cryptographic keys
+
+## Applicability to Client and Server Authentication
+
+The `cmw_attestation` extension is applicable to both client and server authentication in Exported Authenticator-based post-handshake authentication.
+
+### Client Authentication
+For client authentication, the server acts as the relying party and requests attestation from the client. The client may respond with either:
+
+- Evidence (Passport Model):
+  - The client generates Evidence and includes it in the cmw_attestation extension.
+  - The server forwards the Evidence to a Verifier for evaluation and waits for an Attestation Result.
+  - The server grants or denies access based on the Attestation Result.
+
+- Attestation Result (Background Check Model):
+  - The client sends Evidence to a Verifier beforehand.
+  - The Verifier issues an Attestation Result to the client.
+  - The client includes the Attestation Result in cmw_attestation and sends it to the server.
+  - The server verifies the Attestation Result directly, without needing a separate Verifier.
+
+### Server Authentication
+For server authentication, the client acts as the relying party and requests attestation from the server. The server may respond with either:
+
+- Evidence (Passport Model):
+  - The server includes Evidence in cmw_attestation.
+  - The client forwards the Evidence to a Verifier and waits for an Attestation Result.
+  - The client evaluates the attestation result from the Verifier and may choose to continue or terminate the TLS session based on the assessment. If the attestation result fails security policy validation, the client will terminate the TLS session.
+
+- Attestation Result (Background Check Model):
+  - The server sends Evidence to a Verifier beforehand.
+  - The Verifier issues an Attestation Result to the server.
+  - The server includes the Attestation Result in cmw_attestation.
+  - The client validates the Attestation Result directly without consulting an external Verifier.
+
+By allowing both Evidence and Attestation Results to be conveyed within cmw_attestation, this mechanism supports flexible attestation workflows based on the chosen trust model.
 
 # Architecture
 
-Designers of application layer protocols need to define payload formats for conveying exported authenticators that contain remote Evidence. They must also provide mechanisms to inform both communication partners of their ability to exchange Evidence and Attestation Results via this specification. This capability can be specified in a profile of this document or dynamically negotiated during protocol exchanges. A future version of this specification will provide more details.
+The cmw_attestation extension enables attestation evidence to be included in the Certificate message during Exported Authenticator-based post-handshake authentication, ensuring that attestation remains bound to the TLS session.
```

Comment:
Similar to the section on `Usage in Post-Handshake Authentication`, this seems to be specific to evidence and I'm not sure if this is on purpose and the passport model is skipped for some unspecified reason? (old text seems to mention both)